digitalocean/digital_ocean_instance.go

```diff
@@ -54,25 +54,20 @@ func (do *DigitalOcean) CreateInstance(ctx *lepton.Context) error {
 	return nil
 }
 
-// GetInstanceByID returns the instance with the id passed by argument if it exists
-func (do *DigitalOcean) GetInstanceByID(ctx *lepton.Context, id string) (droplet *lepton.CloudInstance, err error) {
+// GetInstance returns instance with given name
+func (do *DigitalOcean) GetInstance(ctx *lepton.Context, name string) (*lepton.CloudInstance, error) {
```

Comment:
Heads up that having named returned arguments simplifies the function. Anyway, I think this is a matter of code style.

Reply:
both styles are fine for me, but the reason was the other methods in that file do not use named return values.